settings_vial/settings_vial.py

```diff
@@ -0,0 +1,110 @@
+import json
+import warnings
+from os import environ
+
+from .exceptions import MissingOverrideKeys, NotCallable, UnsupportedSetType
+
+
+class Settings:
+    r"""Settings from environment variables.
+
+    :param env_prefix: prefix to be used when looking through the environment for variables
+    :param override_prefix: prefix to be used when looking for overrideable variables
+    :param override_callable: callable to be used when retrieving the set of keys to be overriden
```

Comment:
Maybe this could be something like: "callable returning a set of keys to be used as overrides."

Reply:
I'll go for `override_keys_function`, I agree it's more intuitive.
Also updated the docstring